project/profiles/capacity4more/modules/c4m/user/c4m_user_notifications/includes/c4m_user_notifications_handlers.inc

```diff
@@ -500,7 +500,7 @@ function c4m_user_notifications_notify_admins_admin_granted($gid, $uid) {
 }
 
 /**
- * Email notification to user about user him being revoked admin status.
+ * Email notification to user about user being revoked admin status.
  */
 function c4m_user_notifications_notify_user_admin_demoted($gid, $uid) {
```

Comment:
Also, why do we need to call this one and not just calling `_c4m_user_notify_user_admin_demoted()` directly ?